test/netlib/mat_convert.py

```diff
@@ -0,0 +1,68 @@
+import scipy.io
+import os
```

Comment:
could you also add a comment of why this script is needed and what it is doing? 
Maybe a README in `/test/netlib/`

Reply:
I refactored and moved the script to the appropriate location.